src/azure-cli/azure/cli/command_modules/backup/tests/latest/preparers.py

```diff
@@ -395,7 +395,7 @@ def _get_key_url(self, **kwargs):
 class AFSPolicyPreparer(AbstractPreparer, SingleValueReplacer):
```

Comment:
1. We don't need `instant_rp_days` anymore?
2. The code you've added for creating a new AFS policy seems fine, but it looks like the AFSPolicyPreparer already existed and presumably is used for some tests. Are you sure that your changes will not break other tests?